src/main/java/reposense/parser/CsvParser.java

```diff
@@ -27,6 +27,7 @@
     protected static final Logger logger = LogsManager.getLogger(CsvParser.class);
 
     private static final String OVERRIDE_KEYWORD = "override:";
+    private static final String EMPTY_LINE_STRING_REPRESENTATION = "[EMPTY LINE]";
```

Comment:
since its going to be displayed as a message, can just name it `MESSAGE_EMPTY_LINE`

Reply:
Done.